README.md

```diff
@@ -1 +1,65 @@
 # org
+
+This repository contains company-wide guidelines on writing code and organizing
+workflow. If Kowainik member don't know how some part of his/her work should be
+he/she should refer to this guide. If some part of workflow is not clear from
+this document — open issue to this repository. But, of course, everyone should
+read this README first.
+
+## Workflow
+
+### Version control system
+
+Git version control system based on GitHub is used.
+
+> TODO: experiment with Pijul at some moment in future.
+
+#### Issues and branches
+
+1. Work should be done under issue. If such issue doesn't exist: create it.
```

Comment:
That's not clear enough. I guess saying *under corresponding github issue*.. The same goes for the line below